js/stores/simulation-store.ts

```diff
@@ -85,6 +85,8 @@ export class SimulationStore {
     if (config.modelId) {
       this.loadCloudModel(config.modelId);
     }
+    // For debugging purposes
```

Comment:
Did you want to keep this in?  I have no opinion, just checking incase you wanted to remove it.

Reply:
Thanks, yes, actually it was intentional. There's something similar in web worker - it exports model instance. It gets useful quite often.